include/wx/msw/missing.h

```diff
@@ -23,7 +23,35 @@
 #endif
 
 #ifndef WM_PRINTCLIENT
-    #define WM_PRINTCLIENT 0x318
+    #define WM_PRINTCLIENT      0x0318
+#endif
+
+#ifndef WM_DPICHANGED
+    #define WM_DPICHANGED       0x02E0
+#endif
+
+#if !defined(_DPI_AWARENESS_CONTEXTS_)
+// From windef.h, 10.0.14393.0
+#define _DPI_AWARENESS_CONTEXTS_
+
+DECLARE_HANDLE(DPI_AWARENESS_CONTEXT);
```

Comment:
We use `WX_MSW_DECLARE_HANDLE` in `wx/defs.h`, so, again, just for consistency, I'd use it here too.